src/js/angular/ttyg/directives/chat-panel.directive.js

```diff
@@ -121,6 +112,14 @@ function ChatPanelComponent(toastr, $translate, TTYGContextService) {
                 }
             };
 
+            $scope.onAskHowDeliveredAnswer = () => {
+                const askHowDerivedAnswerChatItem = getEmptyChatItem();
+                askHowDerivedAnswerChatItem.setQuestionMessage($translate.instant('ttyg.chat_panel.btn.derive_answer.label'));
+                $scope.askingChatItem = cloneDeep(askHowDerivedAnswerChatItem);
```

Comment:
Why the clone was needed? Isn't this a new instance you just created?

Reply:
Removed